panel/src/components/Forms/Previews/BubblesFieldPreview.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="k-bubbles-field-preview" :class="$options.class">
+    <k-bubbles :bubbles="bubbles" />
+  </div>
+</template>
+
+<script>
+import FieldPreview from "@/mixins/forms/fieldPreview.js";
+
+export default {
+  inheritAttrs: false,
+  mixins: [FieldPreview],
+  props: {
+    value: [Array, String]
+  },
+  computed: {
+    bubbles() {
+      let bubbles = this.value;
+
+      // predefined options
+      const options = this.column.options || this.field.options || [];
+
+      if (typeof bubbles === "string") {
+        bubbles = bubbles.split(",");
+      }
+
+      return bubbles.map((bubble) => {
+        if (typeof bubble === "string") {
+          bubble = {
+            value: bubble,
+            text: bubble
+          };
+        }
+
+        for (const option of options) {
+          if (option.value === bubble.value) {
+            bubble.text = bubble.text;
```

Comment:
What does this do?

Reply:
The bubbles field preview is used for select, radio, checkboxes and multiselect fields. This loop translates plain values into proper text if available. I.e. 

```yaml
options: 
  value: a
  text: Option A
```  

The value which the preview receives might be `a`, but it will make sure to show `Option A` instead with this loop.

Reply:
@bastianallgeier but shouldn't it then be `bubble.text = option.text`?

Reply:
Oh, ok. I'm obviously blind.

Reply:
✅